src/pages/UserHome.jsx

```diff
@@ -109,35 +107,40 @@ export default function UserHome() {
             Add Scholarship
           </Button>
         </Grid>
+        {scholarships.length === 0 && (
```

Comment:
This makes this show up when scholarships are loading. 

https://user-images.githubusercontent.com/8023233/110750375-39b4fd80-81f7-11eb-89ea-088a711dac04.mp4

If there's an error or scholarships are loading this shouldn't show up. Look at how `ScholarshipList` is conditionally loaded below for an example on how to do this. In fact, this should probably go there

Reply:
What you want to do is, just like the actual scholarships themselves below, only show the whole "no scholarships added yet" thing when there are no errors and nothing is loading.